src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
+Unfortunately, obtaining this information currently often relies on online tracking, around which [many large and small companies have built their business](https://crackedlabs.org/en/corporate-surveillance/). This has even led to <a hreflang="en" href="https://www.forbrukerradet.no/wp-content/uploads/2021/06/20210622-final-report-time-to-ban-surveillance-based-advertising.pdf">calls to ban targeted advertising</a>, since invasive tracking is at odds with users' privacy. Users might not want anyone to follow their tracks across the web—especially when visiting websites on sensitive topics. We'll look at the main companies and technologies that make up the online tracking ecosystem.
 
 ### Third-party tracking
 
-Online tracking is often done through certain third-party libraries. These libraries usually provide some (useful) service, but in the process some of them also generate a unique identifier for each user, which can then be used to follow and profile users across websites. The <a hreflang="en" href="https://whotracks.me/">WhoTracksMe</a> project is dedicated to discovering the most widely deployed online trackers. We use WhoTracksMe's classification of trackers, but restrict ourselves to four <a hreflang="en" href="https://whotracks.me/blog/tracker_categories.html">categories</a>: advertising, pornvertising, site analytics and social media.
-
-We see that Google-owned domains are prevalent in the online tracking market, namely in site analytics. Google Analytics, which reports website traffic, is present on almost two thirds of all websites. Around 30% of sites include Facebook libraries, while other trackers only reach single-digit percentages. Overall, 82% of mobile sites and 83% of desktop sites include at least one tracker, usually for site analytics or advertising purposes. Three out of four websites have fewer than 10 trackers, but there is a long tail of sites with many more trackers: one desktop site contacted 133 (!) distinct trackers.
-
+Online tracking is often done through third-party libraries. These libraries usually provide some (useful) service, but in the process some of them also generate a unique identifier for each user, which can then be used to follow and profile users across websites. The <a hreflang="en" href="https://whotracks.me/">WhoTracksMe</a> project is dedicated to discovering the most widely deployed online trackers. We use WhoTracksMe's classification of trackers but restrict ourselves to four <a hreflang="en" href="https://whotracks.me/blog/tracker_categories.html">categories</a>: _advertising_, _pornvertising_, _site analytics_ and _social media_.
 
+{# TODO - Can you explain _why_ you are restricting yourselves to just those four categories? #}
```

Comment:
Suggestion to add an explanation here as to _why_ you are restricting yourselves to just those four categories?

Reply:
see d87c1a4